docs/mllib-evaluation-metrics.md

```diff
@@ -461,11 +461,11 @@ $$rel_D(r) = \begin{cases}1 & \text{if $r \in D$}, \\ 0 & \text{otherwise}.\end{
     <tr>
       <td>Normalized Discounted Cumulative Gain</td>
       <td>
-        $NDCG(k)=\frac{1}{M} \sum_{i=0}^{M-1} {\frac{1}{IDCG(D_i, k)}\sum_{j=0}^{n-1}
+        $NDCG(k)=\frac{1}{M} \sum_{i=0}^{M-1} {\frac{1}{IDCG(D_i, k)}\sum_{j=1}^{n}
```

Comment:
We do need to fix this, but, this makes the subscripts incorrect for R_i(j). I think the expression should change to ln(j+2) in the next line; this is what the code does. For consistency I'd do the same below too.